src/minimega/vmlist.go

```diff
@@ -190,7 +190,9 @@ func (vms VMs) launch(name string, vmType VMType, ack chan int) error {
 	vms[vm.GetID()] = vm
 	vmLock.Unlock()
 
-	return vm.Launch(ack)
+	go vm.Launch(ack)
```

Comment:
This was broken before and is continued here. We should be able to get an error back to minicli on a launch error if we're not using noblock. In this design and in the previous design launch errors just vanish unless you're on a local terminal and see the error log come out on stderr.

The comment above this function also states that the function should block until the launch is complete. That's no longer true here, though it probably should be, with the specific implementation's Launch() returning and error, and concurrency/blocking occuring in the cli/upper level call (which is half-does now).